dashboard/lib/pd/foorm/foorm_parser.rb

```diff
@@ -1,28 +1,34 @@
-# Parses Foorm forms into a useful format for looking up questions by form name, form version
-# and question name
+# Parses Foorm forms into a useful format for looking up questions by type (general or facilitator),
+# form name, form version and question name. General questions are for the workshop overall, facilitator
+# questions are asked on a per-facilitator basis
 # @return
 #   {
-#     <form-name>.<form-version: {
-#       <question_name>: {
-#         title: "sample title",
-#         type: "text/singleSelect/multiSelect/matrix/scale",
-#         # for singleSelect/multiSelect/scale
-#         choices: {
-#           choice_1_name: "choice 1 value",
-#           ...
-#         },
-#         # if has other choice
-#         other_text: "Other choice text",
-#         # for matrix
-#         rows: {
-#           row_1_name: "row 1 value",
-#           ...
-#         },
-#         columns: {
-#           column_1_name: "column 1 value",
-#           ...
+#     general: {
+#       <form-name>.<form-version>: {
+#         <question_name>: {
+#           title: "sample title",
+#           type: "text/singleSelect/multiSelect/matrix/scale",
+#           # for singleSelect/multiSelect/scale
+#           choices: {
+#             choice_1_name: "choice 1 value",
+#             ...
+#           },
+#           # if has other choice
+#           other_text: "Other choice text",
+#           # for matrix
+#           rows: {
+#             row_1_name: "row 1 value",
+#             ...
+#           },
+#           columns: {
+#             column_1_name: "column 1 value",
+#             ...
+#           }
 #         }
 #       }
+#     },
+#     facilitator: {
+#       <form-name>.<form-version>: {...}
 #     }
```

Comment:
Is there supposed to be anything else in here?

Reply:
I put {...} to signify it's the same format as general, I'll update to make that clearer